k2/python/k2/version.py

```diff
@@ -7,6 +7,11 @@
 import torch  # noqa
 import _k2
 
+__build_type__ = _k2.version.build_type
```

Comment:
Do we need other attributes?

Reply:
I think this is enough-- thanks!
I'll wait for someone else to add related features to snowfall though, e.g. to print such info at the start of logs.